packages/cardpay-sdk/sdk/utils/module-utils.ts

```diff
@@ -26,7 +22,7 @@ export async function deployAndSetUpModule(web3: Web3, masterCopy: Contract, set
 
   let encodeInitParams = web3.eth.abi.encodeParameters(setupArgs.types, setupArgs.values);
   let moduleSetupData = masterCopy.methods.setUp(encodeInitParams).encodeABI();
-  let saltNonce = new Date().getTime().toString();
+  let saltNonce = String(getRandomInt(1000000));
```

Comment:
@habdelra I updated the saltNonce value with random integer, it is not possible to use UUID V4 as the value of saltNonce because the data type of saltNonce in GnosisSafeProxFactorty and ModuleProxyFactory is uint256.

Reply:
a uuid v4 has 128 bits. a random number between 1 and 1000000 has 20 bits. we might wanna have a much larger random number to prevent collisions, or perhaps concat the random number to the current time.

Reply:
another idea is to convert a uuid v4 string to it's code points (aka char codes) which are numbers.

Reply:
we also might wanna think about namespacing too. if we decide to make safes for some other thing that uses create2, that will prevent the safes from colliding. like a prefix of "cardstack-scheduled-payments" in front of the UUID beefore we convert it into code points.

Reply:
Got out of range error when using uuidv4 codepoint as saltNonce

Reply:
did you get the out of range in solidity or javascript--I think that number above can fit into uint256. in javascript you need to treat this as string concat and not integer math (like my example above shows).

Reply:
I just checked and the max uint256 looks like 115792089237316195423570985008687907853269984665640564039457584007913129639935, which looks like  4 decimal points larger than the example I posted above

Reply:
I got that error in javascript because the library function like `ethers.utils.solidityKeccak256` will treat uint256 as BigNumber and it causes the out-of-range error. I need to use that library function to calculate the module proxy address.

Reply:
that seems like a bug--since this is a legit uint256 number. can you try truncating the number by a single digit at a time and see what the largest number is that works? (in terms of how many digits you can use)

Reply:
ok, that actually makes sense since 78 digits could be larger than max uint256. let's just truncate to to 77 digits and I think we'll be ok.

Reply:
updated